app/assets/stylesheets/components/_button.scss

```diff
@@ -0,0 +1,25 @@
+.btn {
+  display: flex;
+  width: 264px;
+  padding: 12px 16px;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  align-self: stretch;
+  border-radius: 12px;
+  color: white;
```

Comment:
tu peux faire une variable plutôt pour les couleurs stp ?

Reply:
it's done 👍